### ISSUE TITLE
rflann version

### ISSUE THREAD

**Issue by MjelleLab** [fabotao/GapClust#1]
ERROR: dependency ‘rflann’ is not available for package ‘GapClust’
* removing ‘/Library/Frameworks/R.framework/Versions/4.0/Resources/library/GapClust’
Warning messages:
1: package ‘rflann’ is not available (for R version 4.0.2) 
2: In i.p(...) :
  installation of package ‘/var/folders/z5/fnjct4t12nv2_w0qv5k2m9jm0000gn/T//RtmpeFoPqw/file39a710660102/GapClust_0.1.0.tar.gz’ had non-zero exit status

**Comment by poseidonchan** [comment]
I have the same issue, it seems that rflann is not under maintenance and is not updated for a long time...

**Comment by fabotao** [comment]
Yeah, I have noticed that the rflann packages is not updated for long time. Still it can be installed with install_github function if the R version is OK. Moreover, the PCA analysis done with other libraries also works if adjust a little. I will fixed this soon.
Thanks for your advice. Hope this helps.

----- 原始邮件 -----
发件人：Yanshuo Chen ***@***.***>
收件人：fabotao/GapClust ***@***.***>
抄送人：Subscribed ***@***.***>
主题：Re: [fabotao/GapClust] rflann version (Issue #1)
日期：2022年10月16日 00点05分


I have the same issue, it seems that rflann is not under maintenance and is not updated for a long time...
